frontend/src/scenes/events/EventsTable.tsx

```diff
@@ -298,6 +301,10 @@ export function EventsTable({ fixedFilters, filtersEnabled = true, pageKey }: Ev
                 defaultColumns={defaultColumns.map((e) => e.key || '')}
                 onColumnUpdate={setColumnConfig}
                 saving={columnConfigSaving}
+                automaticLoading={{
+                    toggle: toggleAutomaticLoad,
+                    enabled: automaticLoadEnabled,
+                }}
```

Comment:
I guess this is fine, but so far I haven't encountered a pattern where we pass a `value` and it's `onChange` (well, `toggle` and `enabled`) in a combined object prop like this. Here though, I'd personally skip the extra destructuring and just pass this as two separate props. 

However. There is a lot of smelly code here. The pattern that we normally use is to run `useValues` and `useActions` and directly fetch the [data](https://kea.js.org/blog/2021/05/14/data-first-frontend-revolution) from a logic. I'd suggest doing it here as well, but somehow this `TableConfig` is an abstract library component, making it hard to do so. Instead I'd reevaluate why this is library component in the first place, and then just move it into the `scenes/events` folder, in order to get the data from the scene directly.

Reply:
Yep, I like giving things structure/names but was wondering if it was a signal the thing was in the wrong place.

There are four uses of resizeable table but none use it the same way as the EventsTable so it _should_ (famous last words) be relatively easy to move the cleverness up